newsroom/companies/companies.py

```diff
@@ -46,7 +46,19 @@ class CompaniesResource(newsroom.Resource):
         "allowed_ip_list": {"type": "list", "mapping": {"type": "string"}},
         "original_creator": newsroom.Resource.rel("users"),
         "version_creator": newsroom.Resource.rel("users"),
+        "products": {
+            "type": "list",
+            "schema": {
+                "type": "dict",
+                "schema": {
+                    "_id": newsroom.Resource.rel("products"),
+                    "seats": {"type": "number"},
```

Comment:
Should this have a default of `0`, so this attribute is optional but always has a value?